### PR TITLE
Gradient (AltTo) and increased the size to look good. 

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
Hi Madis0, remember me? 

### For Live Example:

My fork of FO: https://github.com/PSGitHubUser1/fabulously-optimized

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,17 +4,19 @@ A Fabric-based Minecraft modpack focusing on performance and graphics enhancemen
...
+
+<a href="https://download.fo/mastodon"><img alt="Mastodon" height="60" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/social/mastodon-plural_vector.svg"></a>&nbsp;
 
 ## Downloads
 
 [![Download from CurseForge](https://cf.way2muchnoise.eu/full_fabulously-optimized_downloads%20on%20CurseForge.svg?badge_style=for_the_badge)](https://download.fo/curseforge) [![Modrinth](https://img.shields.io/modrinth/dt/fabulously-optimized?logo=modrinth&color=darkgreen&label=Download%20from%20Modrinth&style=for-the-badge)](https://download.fo/modrinth)
-<a href="https://download.fo/install"><img alt="How to install" height="28" src="https://github.com/Fabulously-Optimized/fabulously-optimized/assets/8611110/0aaed593-9791-4d24-8a64-8552e6bffa07"></a>
+<a href="https://download.fo/install"><img alt="How to install" height="28" src="https://github.com/Fabulously-Optimized/fabulously-optimized/assets/8611110/0aaed593-9791-4d24-8a64-8![compact_46h](https://github.com/PSGitHubUser1/fabulously-optimized/assets/90406016/fe3ed69a-a95a-4b30-b7be-506e09cd9a2f)
```
Comment (Madis0) [inline README.md:18]: That is broken

File: README.md [hunk 2]
```diff
@@ -4,17 +4,19 @@ A Fabric-based Minecraft modpack focusing on performance and graphics enhancemen
...
-<a href="https://download.fo/rate"><img alt="Rate" height="40" src="https://github.com/Fabulously-Optimized/fabulously-optimized/assets/8611110/fce9cc22-36a7-4dad-89cf-b47a2f08daa8"></a>
+<a href="https://wiki.download.fo/"><img alt="Wiki" height="60" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/documentation/gitbook_vector.svg"></a>&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="https://download.fo/translate"><img alt="Translate" height="60" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/translate/generic-plural_vector.svg"></a>
 
-<a href="https://download.fo/discord"><img alt="Discord" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/social/discord-plural_vector.svg"></a> 
-<a href="https://download.fo/mastodon"><img alt="Mastodon" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/social/mastodon-plural_vector.svg"></a>
+<a href="https://download.fo/rate"><img height="60" src="https://github.com/PSGitHubUser1/fabulously-optimized/assets/90406016/c2b91fcf-0d7c-41ec-8af6-b4a5671c8bed"></a>&nbsp;&nbsp;
```
Comment (Madis0) [inline README.md:10]: Try to view it on mobile, it appears stretched

**Review by Madis0 — CHANGES_REQUESTED** [review]

**Comment by PSGitHubUser1** [comment]
Is it better now??

File: README.md [hunk 3]
```diff
@@ -4,12 +4,13 @@ A Fabric-based Minecraft modpack focusing on performance and graphics enhancemen
 
 [**📜 Mod list**](INCLUDED-MODS.md) | [**🎛️ Changed options**](https://wiki.download.fo/readme/changed-options) | [**❤️ Support and contribute**](https://download.fo/thanks)
 
-<a href="https://wiki.download.fo/"><img alt="Wiki" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/documentation/gitbook_vector.svg"></a> 
-<a href="https://download.fo/translate"><img alt="Translate" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/translate/generic-plural_vector.svg"></a>
-<a href="https://download.fo/rate"><img alt="Rate" height="40" src="https://github.com/Fabulously-Optimized/fabulously-optimized/assets/8611110/fce9cc22-36a7-4dad-89cf-b47a2f08daa8"></a>
+<a href="https://wiki.download.fo/"><img alt="Wiki" height="50" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/documentation/gitbook_vector.svg"></a>&nbsp;&nbsp;&nbsp;
```
Comment (Madis0) [inline README.md:7]: Checked on desktop, the increased badge size seems too big to me, current size was fine.

Comment (Madis0) [inline README.md:7]: The spaces `&nbsp;` make the badges weirdly aligned on mobile.

**Review by Madis0 — CHANGES_REQUESTED** [review]

**Comment by Madis0** [comment]
Merged the gradient change. Thanks!

**Pull request merged**